files/default.conf

```diff
@@ -40,6 +40,12 @@ server {
     root /var/www/html;
     index index.html;
 
+    add_header Content-Security-Policy "default-src *; style-src 'self' 'unsafe-inline'";
```

Comment:
This is very permissive and probably could be tightened up a lot, but the support page is not a web application and never displays user-generated content. Let's not spend more time on it.